docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1099,15 +1082,15 @@ fail! Here's a few:
 
 With Cypress, you don't have to assert to have a useful test. Even without
```

Comment:
I feel like this sentence could maybe use a couple qualifiers, though I do see you're trying to make things less wordy:

```suggestion
With Cypress, you don't always have to explicitly assert to have a useful test. Even without
```

Again, it wasn't written in this PR, so feel free to ignore.